file_organizer_v2/docs/phase4/analytics.md

```
@@ -0,0 +1,793 @@
+# Get storage statistics
+storage = analytics.get_storage_stats(Path("~/Documents"))
+
+print(f"Total Size: {storage.formatted_total_size}")
+print(f"Files: {storage.file_count}")
+print(f"Directories: {storage.directory_count}")
+print(f"Average file size: {storage.avg_file_size}")
+```
+
+### Size Distribution
+
+**By File Type**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)